src/Effectful/HPQTypes.hs

```diff
@@ -29,6 +29,10 @@ import Effectful.Dispatch.Dynamic
 import Effectful.Error.Static
 import Effectful.State.Static.Local
 
+-- | An effect for the hpqtypes bindings to libpqtypes
+--
+-- An `Eff es` stack that contains `EffectDB` can use all functions
```

Comment:
as above, we could replace "can use" with "allows to use"

Reply:
done